projects/packages/my-jetpack/_inc/components/product-card/index.test.jsx

```diff
@@ -1,31 +0,0 @@
-import { composeStories } from '@storybook/react';
```

Comment:
We don't add any tests instead?

Reply:
I don't think it's necessary to add tests, at least to replace these ones. `onManage` and `onAdd` have been removed in favor of using `href`, which ultimately is passed to an `a` tag.